addons/binding/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/SomfyTahomaBindingConstants.java

```diff
@@ -188,6 +192,7 @@
     final public static String HEATINGSYSTEM = "HeatingSystem";
     final public static String DOORLOCK = "DoorLock";
     final public static String PERGOLA = "Pergola";
+    final public static String WINDOWHANDLE = "WindowHandle";
```

Comment:
The correct order is `public static final` please update everywhere in the file :-)

Reply:
Fixed